src/sass/style.scss

```diff
@@ -44,3 +44,9 @@
 /* Trumps           */
 /********************/
 
+// Temporary work-around for bug in Gravity's
+// column layout styling that causes the page to
+// overflow the viewport horizontally.
+.grav-o-container {
+  overflow-x: hidden;
```

Comment:
This isn't lovely 😆 

Not going to block the merge for it though

Reply:
I know - far from it.

Thanks for approving though!